src/components/standalone/users_database/UsersDatabaseManager.vue

```diff
@@ -70,6 +71,14 @@ function openDeleteUserModal(itemToDelete: User) {
   selectedUser.value = itemToDelete
   showDeleteUserModal.value = true
 }
+async function setAdmin(item: User) {
+  await ubusCall('ns.users', 'set-admin', { name: item.name, database: 'main' })
+  refreshUsers()
+}
+async function removeAdmin(item: User) {
+  await ubusCall('ns.users', 'remove-admin', { name: item.name, database: 'main' })
+  refreshUsers()
+}
```

Comment:
Are there situations where an error can be returned?

Reply:
Worry not! Was just asking if a forseeable error might be returned, since it's not the case, we are all good!